### DISCUSSION TITLE
Tasmotized Sonoffs don't connect to AP over WiFi N (lite, as there is only 1 antenna)

### DISCUSSION THREAD

**Discussion by laurentppol** [arendst/Tasmota#15451] [Q&A] [answered]
Hi there, 
I observed this (in topic) using an OpenWRT'ized (18.06) TP-LINK WR-1043ND.
ALL my Tasmotized (Sonoffs) devices connect using b/g 802.11 standard. No more than 54Mbps (channel 20MHz).
No MCS, etc.
I have 2 devices NOT Tasmota based:
- Sonoff RF Bridge R2 - ITEAD firmware, if Tasmotized, connects as b/g, 
- my home made thermomether (using quite old Sming framework) on NodeMCU v2.

My other devices (also TP-LINKS) connect with rates approaching 135Mbps.

And those devices connect to AP using different rates, sometimes fractional, and with MCS.

So problem seems to be related to Arduino framework.
If needed some more info, please tell.

**Comment by ascillato** [comment, marked as answer]
Wi-Fi Mode N is supported in Arduino, hence it is supported in Tasmota. Please, try command `wifi 4` in the console to enable b/g/n. Thanks.

https://tasmota.github.io/docs/Commands/#wi-fi

By default, Tasmota have all modes enabled, but if the device was flashed without erasing first, or your device have been updated from an old Tasmota version without following the upgrade path, or the command `wifi` has been used to disable modes, your device won't be able to connect to your desired mode.

**Comment by lalo-uy** [comment]
From which version on do wifi command exist?

I got :
wget -q -T2 -t2 -O - http://192.168.100.9/cm?cmnd=wifi
{"Command":"Unknown"}

on "Version":"8.1.0(lite)"

**Reply by sfromis** [reply]
Don't even ask when your version is that badly outdated. Just upgrade :wink:
I think that the `Wifi` command came with version 9.5.

Note that not all current options of the command were there from the start.
